ramls/edge-patron.raml

```diff
@@ -30,9 +30,8 @@ types:
       description: Service endpoints that manage accounts by an existing Id
       uriParameters:
         id:
-          description: The UUID of a FOLIO user
+          description: Patron's external system Id stored in FOLIO user record.
           type: string
-          pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
```

Comment:
Please update the minor interface version to reflect this change

Reply:
That would be the version number in the RAML file, correct?

Reply:
And in the module descriptor if there is one